interface/patient_file/front_payment.php

```diff
@@ -12,13 +12,13 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
-require_once("../globals.php");
-require_once("$srcdir/patient.inc.php");
-require_once("$srcdir/payment.inc.php");
-require_once("$srcdir/forms.inc.php");
-require_once("../../custom/code_types.inc.php");
-require_once("$srcdir/options.inc.php");
-require_once("$srcdir/encounter_events.inc.php");
+require_once "../globals.php";
+require_once "$srcdir/patient.inc";
+require_once "$srcdir/payment.inc.php";
+require_once "$srcdir/forms.inc";
+require_once "../../custom/code_types.inc.php";
+require_once "$srcdir/options.inc.php";
+require_once "$srcdir/encounter_events.inc.php";
```

Comment:
revert above block of code to original

Reply:
(if the issue is that your current environment is missing the new *.inc.php files (ie. you are not working in most recent codebase), then you can get these scripts from master and put them in your current environment)